doc/developers/contributing.rst

```diff
@@ -952,13 +952,14 @@ The easiest and recommended way to accomplish this is to
 All logic behind estimator parameters,
 like translating string arguments into functions, should be done in ``fit``.
 
-Parameters can be passed to the ``fit`` method. Generally, these parameters
+While it is possible to pass parameters into the ``fit`` method, these
 should be restricted to variables that need to be sliced during
-cross-validation. These variables would be either arrays with shape=[N] or
-array-like with shape=[N,] where N is the number of samples. An example of
+cross-validation. These variables would be either arrays with shape=[N] where N=n_samples
```

Comment:
I think we only require ``shape[0] == n_samples``. Why not just ``These would be arrays with ``shape[0] == n_samples``?